src/accumulator/stump.rs

```diff
@@ -0,0 +1,139 @@
+use super::types;
```

Comment:
We should stick to 4 spaces for indentation

Reply:
Regarding code style, I ran `rustfmt` into the code, so it now conforms  rust's code standards. I think it's ok now.

Reply:
We should run it on the other files as well.. Should add it to the CI

Reply:
> We should run it on the other files as well  

I didn't run for this commit because those files are out of scope for this PR. But 100% agree in running `rustfmt` for everything else.  

> Should add it to the CI  

AFAIK, this codebase has no CI, this is a nice TODO.